modules/ROOT/pages/streaming.adoc

```diff
@@ -12,6 +12,36 @@ Streaming supports efficient processing of large data objects such as files, doc
 * Simple routing rules based on message metadata are still possible
 * You can combine streaming and non-streaming endpoints
 
+== Data Streams Consumption
+To understand how data streams are consumed, review the following points:
+
+* Data streams cannot be consumed more than once
++
+In the following example, the flow writes the first file correctly, while the second file is created with empty content because each component that consumes a stream expects to receive a new stream. After the first File Write operation consumes the stream, the second File Write operation receives an empty stream. Thereby, the second operation has no content to write to a file.
++
+.Mule 3 Streaming: Writing a File
+image::mruntime-streaming-about-1.png[]
++
+In the following example, something similar happens when you try to log the payload before and
+after a DataWeave transformation. The flow logs the payload before the Transform Message component, but does not
+log the resulting payload after the Transform Message component because the Logger component consumes
+and loads the stream into memory.
```

Comment:
i don't understand what you mean by `loads the stream into memory ` here, having it in memory would imply that it is not consumed, it is confusing

Reply:
Removed that part.